### PR TITLE
Defocus download button after clicking

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Review by inodb — APPROVED** [review]

**Comment by onursumer** [comment]
@adamabeshouse Do you know why the build is failing?

File: components/FileTable.tsx [hunk 1]
```diff
@@ -209,8 +209,10 @@ export default class FileTable extends React.Component<IFileTableProps> {
         makeObservable(this);
     }
 
-    @action onDownload = () => {
+    @action onDownload = (evt:any) => {
         this.isDownloadModalOpen = true;
+        // Defocus "Download" button
+        evt.target.blur();
```
Comment (alisman) [inline components/FileTable.tsx:215]: what does this fix?

Comment (adamabeshouse) [reply components/FileTable.tsx:215]: its a UI request from niki

**Comment by alisman** [comment]
I think this is kind of silly.  It's the default behavior of buttons in browser and has something to do with accessibility.  I would be in favor of not doing this

**Pull request closed without merging**